src/MessagingInput/MessagingInput.less

```diff
@@ -83,25 +90,7 @@
   margin-left: 0.0625rem; // 1 px because the FA icon is slightly off center
 }
 
-.MessagingInput--TextFieldLabel {
-  color: @neutral_medium_gray;
-  .margin--bottom--2xs();
```

Comment:
I removed this margin bottom and added it to the parent container instead, so the margin would be the same on the `Label` and the `Checkbox`, both which live within `TopInfo`